utils/src/ast-grep/resolve-binding-path.test.ts

```diff
@@ -0,0 +1,151 @@
+import assert from "node:assert/strict";
+import { resolveBindingPath } from "./resolve-binding-path.ts";
+import { describe, it, only } from "node:test";
+import astGrep from "@ast-grep/napi";
+import dedent from "dedent";
```

Comment:
```suggestion
import assert from "node:assert/strict";
import { describe, it, only } from "node:test";
import astGrep from "@ast-grep/napi";
import dedent from "dedent";
import { resolveBindingPath } from "./resolve-binding-path.ts";
```

my import style guide

- node 
- npm apckage
- relative path 
- type import with same order